### PR TITLE
Add CeloDance Wallet to Celo use-contractkit

### PR DESCRIPTION
**Pull request by bi23com**
as a celo mobile wallet, celodance support the transfer and staking of celo assets. after talking with clabs, we propose to integrate celodance wallet to moola and ubeswap. the development work are all finished, so please merge the code, thanks!

### CODE REVIEW

**Comment by jmrossy** [comment]
Note this is a recreation of PR https://github.com/celo-tools/use-contractkit/pull/33
I will review it later today

**Comment by jmrossy** [comment]
@bi23com This updated version looks good to me! Please just add a screenshot of the updated version to this PR so I can confirm everything looks right before we merge.

**Comment by bi23com** [comment]
WechatIMG341
WechatIMG342
WechatIMG343

hi, I just uploaded the screenshots of our new function. please check it out! thank you! you can click the button in the upper right to scan the QR code , once you're connectted,  you can use celodance wallet to deposit/ borrow/swap in Moola and ubeswap.

**Comment by jmrossy** [comment]
@bi23com Thanks for the screenshots but sorry if I wasn't clear. Please share pictures of **use-contractkit** since that's what's being changed in this PR :)

**Comment by bi23com** [comment]
WechatIMG344
WechatIMG345

Sorry for the trouble. Please check it out.many thanks！

**Comment by jmrossy** [comment]
@bi23com Why do you keep closing PRs and opening new ones? Please just reopen this one and push your edits to this existing PR.

File: packages/use-contractkit/src/constants.tsx [hunk 1]
```diff
@@ -53,6 +55,15 @@ export const PROVIDERS: {
     showInList: () => true,
     listPriority: () => 0,
   },
+  [SupportedProviders.CeloDance]: {
```
Comment (jmrossy) [inline packages/use-contractkit/src/constants.tsx:58]: Please move this down below Metamask. Additions added later should not be given high placement in the list, sorry!

**Review by jmrossy — COMMENTED** [review]
One comment to address

**Comment by bi23com** [comment]
hi Rossy，sorry for the trouble, we close the #36 and #37 and reopen this one, please review, thank you!

**Pull request merged**